src/main/java/org/springframework/hateoas/client/Rels.java

```diff
@@ -103,6 +102,10 @@ public Link findInResponse(String response, MediaType mediaType) {
 
 			return discoverer.findLinkWithRel(rel, response);
 		}
+
+		public String toString() {
```

Comment:
Make sure this gets and `@Override` and the obligatory `(non-Javadoc) @see` multi-line comment for consistency.